src/main/java/com/avispl/symphony/dal/communicator/lumen/vc/tr60a/LumenVCTR60ACommunicator.java

```diff
@@ -0,0 +1,1985 @@
+	 * @param currentPropertyName is the label for current value of property
+	 * @param propertyValue is the current value of property
+	 * @param labelStart is the label start of slider
+	 * @param labelEnd is the label end of slider
+	 * @param rangeStart is the range start of slider
+	 * @param rangeEnd is the range end of slider
+	 * @param initialValue is the initial value of slider
+	 */
+	private void populateSliderControl(Map<String, String> stats, List<AdvancedControllableProperty> advancedControllableProperties, String propertyName,
+			String currentPropertyName, String propertyValue, String labelStart, String labelEnd, float rangeStart, float rangeEnd, float initialValue) {
+
+		if (Objects.equals(propertyValue, LumenVCTR60AConstants.NONE_VALUE)) {
+			stats.put(propertyName, LumenVCTR60AConstants.NONE_VALUE);
+			stats.put(currentPropertyName, LumenVCTR60AConstants.NONE_VALUE);
+			return;
+		}
+		stats.put(propertyName, String.valueOf(initialValue));
+		stats.put(currentPropertyName, propertyValue);
+		addAdvancedControlProperties(advancedControllableProperties, stats, createSlider(propertyName, labelStart, labelEnd, rangeStart, rangeEnd, initialValue), LumenVCTR60AConstants.NOT_AVAILABLE);
+	}
+
+	/**
+	 * This method is used for populate switch control
+	 *
+	 * @param stats is the map that store all statistics
+	 * @param advancedControllableProperties is the list that store all controllable properties
+	 * @param propertyName is the property name of switch control
+	 * @param currentStatus is the current status of switch control
+	 * @param labelOff is the label off of switch control
+	 * @param labelOn is the label on of switch control
+	 */
+	private void populateSwitchControl(Map<String, String> stats, List<AdvancedControllableProperty> advancedControllableProperties, String propertyName, String currentStatus,
+			String labelOff, String labelOn) {
+		if (Objects.equals(currentStatus, LumenVCTR60AConstants.NONE_VALUE)) {
+			stats.put(propertyName, LumenVCTR60AConstants.NOT_AVAILABLE);
+			return;
+		}
+
+		if (Objects.equals(currentStatus, labelOn)) {
+			stats.put(propertyName, String.valueOf(1));
+			advancedControllableProperties.add(createSwitch(propertyName, 1, labelOff, labelOn));
+		} else if (Objects.equals(currentStatus, labelOff)) {
+			stats.put(propertyName, String.valueOf(0));
+			advancedControllableProperties.add(createSwitch(propertyName, 0, labelOff, labelOn));
+		}
+	}
+
+	/**
+	 * This method is used for populate button control
+	 *
+	 * @param stats is the map that store all statistics
+	 * @param advancedControllableProperties is the list that store all controllable properties
+	 * @param propertyName is the property name of button control
+	 * @param buttonLabel is the label of button control
+	 */
+	private void populateButtonControl(Map<String, String> stats, List<AdvancedControllableProperty> advancedControllableProperties, String propertyName, String buttonLabel) {
+		stats.put(propertyName, "");
```

Comment:
We should set N/A value in here, so that Empty value isn't shown in historicalProperties tab (when adapter monitoring is down for any reason). That'd be the case for other buttons as well :)